src/current/v23.1/migration-overview.md

```diff
@@ -5,87 +5,304 @@ toc: true
 
-1. The Schema Conversion Tool automatically creates a new {{ site.data.products.serverless }} database with the converted schema. {% include cockroachcloud/migration/sct-self-hosted.md %}
-1. [Move your data](#step-2-move-your-data-to-cockroachdb) to the new database.
+For an overview of zero-downtime migrations to CockroachDB, see [Zero Downtime](#zero-downtime). {% comment %}For details, see [Migration Strategy: Zero Downtime](migration-strategy-zero-downtime).{% endcomment %}
 
-### Schema design best practices
+### Cutover strategy
 
-Consider the following CockroachDB attributes and best practices:
+*Cutover* is the process of switching application traffic from the legacy database to CockroachDB. Consider the following:
 
-- We recommend that you always have an explicit primary key defined on every table. For more information, see [Primary key best practices](schema-design-table.html#primary-key-best-practices).
+- Will you perform the cutover all at once, or incrementally (e.g., by a subset of users, workload, or tables)?
+
+	- Switching all at once generally follows a [downtime window](#downtime-window). Once the data is migrated to CockroachDB, you "flip the switch" to route application traffic to the new database, thus ending downtime.
+
+	- Migrations with [zero or near-zero downtime](#zero-or-near-zero-downtime) can switch either all at once or incrementally, since writes are being synchronously replicated and the system can be gradually migrated as you [validate the queries](#validate-queries).
+
+- Will you have a fallback plan in which you reverse ("roll back") the migration from CockroachDB to the legacy database? A fallback plan enables you to fix any issues or inconsistencies that you encounter during or after cutover, then retry the migration.
+
+#### All at once (no rollback)
+
+This is the simplest cutover plan, since you won't need to develop and execute a fallback method. It is typically used with [lift-and-shift migrations](#lift-and-shift).
+
+After moving all of the data from the legacy database to CockroachDB, you switch application traffic to CockroachDB. Since you cannot roll back the migration after switching over, you **must** [validate your queries](#validate-queries) first.
+
+#### All at once (rollback)
+
+This method can be used for [lift-and-shift migrations](#lift-and-shift), as well as for [zero-downtime migrations](#zero-downtime).
+
+In addition to moving data to CockroachDB, data is also replicated from CockroachDB back to the legacy database. Continuous replication is already possible when performing a zero-downtime migration that dual writes to both databases. Otherwise, you will need to ensure that writes are consistent in both directions.
+
```

Comment:
Do we need to mention that we don't actually support any tools to help with rollback yet? Or not draw the attention to it